src/StudioCore/Assets/Paramdex/ER/Names/BehaviorParam_PC.txt

```diff
@@ -4303,2867 +5326,7148 @@
+106800115 Beast Claw - Default
+106800120 Beast Claw - Default
+106800125 Beast Claw - Default
+106800126 Beast Claw - Default
+106800130 Beast Claw - Default
+106800140 Beast Claw - Default
+106800150 Beast Claw - Default
+106800160 Beast Claw - Default
+106800175 Beast Claw - Default
+106800176 Beast Claw - Default
+106800180 Beast Claw - Default
+106800181 Beast Claw - Default
+106800200 Beast Claw - Default
+106800201 Beast Claw - Default
+106800210 Beast Claw - Default
+106800211 Beast Claw - Default
+106800212 Beast Claw - Default
+106800220 Beast Claw - Default
+106800221 Beast Claw - Default
+106800230 Beast Claw - Default
+106800300 Beast Claw - Default
+106800301 Beast Claw - Default
+106800305 Beast Claw - Default
+106800306 Beast Claw - Default
+106800310 Beast Claw - Default
+106800311 Beast Claw - Default
+106800315 Beast Claw - Default
+106800316 Beast Claw - Default
+106800320 Beast Claw - Default
+106800321 Beast Claw - Default
+106800325 Beast Claw - Default
+106800326 Beast Claw - Default
+106800327 Beast Claw - Default
+106800330 Beast Claw - Default
+106800331 Beast Claw - Default
+106800340 Beast Claw - Default
+106800341 Beast Claw - Default
+106800350 Beast Claw - Default
+106800351 Beast Claw - Default
+106800360 Beast Claw - Default
+106800361 Beast Claw - Default
+106800375 Beast Claw - Default
+106800376 Beast Claw - Default
+106800380 Beast Claw - Default
+106800381 Beast Claw - Default
+106800382 Beast Claw - Default
+106800383 Beast Claw - Default
+106800400 Beast Claw - Default
+106800410 Beast Claw - Default
+106800411 Beast Claw - Default
+106800420 Beast Claw - Default
+106800430 Beast Claw - Default
+106800460 Beast Claw - Default
+106800470 Beast Claw - Default
+106800480 Beast Claw - Default
+106800500 Beast Claw - Default
+106800501 Beast Claw - Default
+106800509 Beast Claw - Default
+106800510 Beast Claw - Default
+106800511 Beast Claw - Default
+106800530 Beast Claw - Default
+106800531 Beast Claw - Default
+106800540 Beast Claw - Default
+106800550 Beast Claw - Default
+106800560 Beast Claw - Default
+106800561 Beast Claw - Default
+106800590 Beast Claw - Default
+106800591 Beast Claw - Default
+106800600 Beast Claw - Default
+106800605 Beast Claw - Default
+106800610 Beast Claw - Default
+106800630 Beast Claw - Default
+106800632 Beast Claw - Default
+106800635 Beast Claw - Default
+106800640 Beast Claw - Default
+106800642 Beast Claw - Default
+106800645 Beast Claw - Default
+106800650 Beast Claw - Default
+106800660 Beast Claw - Default
+106800665 Beast Claw - Default
+106800700 Beast Claw - Default
+106800705 Beast Claw - Default
+106800710 Beast Claw - Default
+106800730 Beast Claw - Default
+106800732 Beast Claw - Default
+106800735 Beast Claw - Default
+106800740 Beast Claw - Default
+106800742 Beast Claw - Default
+106800745 Beast Claw - Default
+106800750 Beast Claw - Default
+106800760 Beast Claw - Default
+106800765 Beast Claw - Default
+106800800 Beast Claw - Default
+106800801
```

Comment:
I'm a fan of this additional tagging for if an AoW row is category specific. Good job.